### PR TITLE
[csharp] Replace ~/ links with ../ relatives

### PR DESCRIPTION
**Pull request by nemrism**
## Summary

Replace `~/` links with `../` relative links in /csharp/** files

Contributes to #9968

### CODE REVIEW

File: docs/fsharp/language-reference/generics/index.md [hunk 1]
```diff
@@ -91,6 +91,6 @@ There are two kinds of type parameters that can be used in F# programs. The firs
 - [Language Reference](../index.md)
 - [Types](../fsharp-types.md)
 - [Statically Resolved Type Parameters](statically-resolved-type-parameters.md)
-- [Generics in the .NET Framework](~/docs/standard/generics/index.md)
+- [Generics in the .NET Framework](../../../standard/generics/index.md)
```
Comment (pkulikov) [inline docs/fsharp/language-reference/generics/index.md]: To match the name used in the C# sections
```suggestion
- [Generics](../../../standard/generics/index.md)
```

Comment (BillWagner) [reply docs/fsharp/language-reference/generics/index.md]: I agree with @pkulikov on this change.

Once you make this, I'll :shipit:

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks again @nemrism 

I agree with @pkulikov 's suggestion. Once you've done that, I'll :shipit:

**Comment by BillWagner** [comment]
@nemrism It looks like this PR also introduces conflicts based on your previous PRs.  Can you rebase this on master?

**Comment by nemrism** [comment]
@BillWagner just pushed the rebase with conflicts resolved

File: docs/fsharp/language-reference/generics/index.md (hunk 1 above)
Comment (BillWagner) [reply docs/fsharp/language-reference/generics/index.md]: I unresolved this conversation. It doesn't appear that it's been made. (Not sure if that's due to the rebase or not)

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @nemrism I've approved these changes, and I'll :shipit: when the build finishes.

**Pull request merged**